js-contribution-guidelines.md

```diff
@@ -0,0 +1,12 @@
+# JS Contribution Guidelines
+
+Many IPFS projects use javascript. Please check these guidelines before contributing js code to an IPFS repository.
+
+## Suggestions
+
+- Build system: Use `npm run` for small projects; for larger ones, use `gulp`.
+- Testing node: `mocha`
+- Testing browser: `karma` + `mocha`
+- Browser building: `webpack` or `browserify` (this fight is still ongoing ;)
```

Comment:
let's not call it a fight, rather point it to issues where discussion is happening and invite folks to collaborate. We really shouldn't even have to lock on either, WebPack is mainly used by front end devs while browserify has a lot of users that are Node.js developers hacking on the browser.